plugin-build/src/test/kotlin/io/sentry/android/gradle/tasks/SentryUploadNativeSymbolsTaskTest.kt

```diff
@@ -0,0 +1,114 @@
+package io.sentry.android.gradle.tasks
+
+import org.gradle.api.Project
+import org.gradle.api.tasks.TaskProvider
+import org.gradle.testfixtures.ProjectBuilder
+import org.junit.Test
+import java.io.File
+import kotlin.test.assertEquals
+import kotlin.test.assertFalse
+import kotlin.test.assertNull
+import kotlin.test.assertTrue
+
+class SentryUploadNativeSymbolsTaskTest {
+
+    @Test
+    fun `cli-executable is set correctly`() {
+        val project = createProject()
+        val task: TaskProvider<SentryUploadNativeSymbolsTask> =
+            project.tasks.register("testUploadNativeSymbols", SentryUploadNativeSymbolsTask::class.java) {
+                it.cliExecutable.set("sentry-cli")
+                it.includeNativeSources.set(false)
+                it.variantName.set("debug")
```

Comment:
maybe we create an aux. method like `createProject` but for the task? this block seems kinda duplicated across tests, if possible I'd extract and use default param values so we reuse it when possible.

Reply:
Yup makes sense 👍